runtime/compiler/rpc/J9Server.cpp

```diff
@@ -66,7 +66,7 @@ J9ServerStream::J9ServerStream(int connfd, uint32_t timeout)
 
 
 void
-J9ServerStream::finishCompilation(uint32_t statusCode, std::string codeCache, std::string dataCache, CHTableCommitData chTableData,
+ServerStream::finishCompilation(uint32_t statusCode, std::string codeCache, std::string dataCache, CHTableCommitData chTableData,
```

Comment:
I propose that we make this function more generic by making it accept a variadic list of template arguments.
This would make adding new arguments easier, and would also remove the need for CHTable import, which is not in OMR.
```
   template <typename... T>
   void finishCompilation(uint32_t statusCode, T... args)
      {
      try
         {
         write(MessageType::compilationCode, statusCode, args...);
         }
      catch (std::exception &e)
         {
         if (TR::Options::getVerboseOption(TR_VerboseJITaaS))
            TR_VerboseLog::writeLineLocked(TR_Vlog_JITaaS, "Could not finish compilation: %s", e.what());
         }
      }
```

Reply:
Could you open an issue so that this thought doesn't get forgotten?

Reply:
Done